klayout/drc/testing/run_regression.py

```diff
@@ -122,11 +122,18 @@ def check_klayout_version():
         exit(1)
     elif len(klayout_v_list) >= 2 or len(klayout_v_list) <= 3:
         if klayout_v_list[1] < 28:
-            logging.error("Prerequisites at a minimum: KLayout 0.28.0")
+            logging.error("Prerequisites at a minimum: KLayout 0.28.4")
```

Comment:
wouldn't this be also done by `run_drc` already?

Reply:
Yes, but we need it also in the regression script to raise this error before starting in regression testing steps. As there are some other steps before calling run_drc script.

Reply:
@proppy Same comment at as above #6

Reply:
Same comment here, this PR is adding a new version check on line 130-137, so it makes sense to review it here.